Project1/TextFile.cs

```diff
@@ -3,26 +3,54 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.IO;
 
 namespace Project1
 {
     internal class TextFile
     {
+        string textFile = @"C:\Users\jarmo\source\repos\Project1\PlikTxt.txt";
         public void ReadAllText()
```

Comment:
Ta metoda jest wgl nei potrzebna. `File.ReadAllLines(textFile);` to metoda wyciągająca wartości do Tablicy. Wystarczy w program.cs po prostu wyświetlić tą tablicę.